src/pcapi/core/providers/api.py

```diff
@@ -196,6 +196,8 @@ def _build_new_offers_from_stock_details(
     for stock_detail in stock_details:
         if stock_detail["offers_provider_reference"] in existing_offers_by_provider_reference:
             continue
+        if not stock_detail["available_quantity"]:
+            continue
```

Comment:
```suggestion
        if stock_detail["available_quantity"] == 0:
            continue
```
?
Je suis pas forcément fan de simplifier par du falsy ici :)

Reply:
Je pense que de toute façon on n'a pas envie de laisser passer une valeur `None` (car ça casserait un peu loin de toute façon quand on additionne `None` au `Stock.dnBookedQuantity`). Donc je garderais bien cette condition telle quelle.